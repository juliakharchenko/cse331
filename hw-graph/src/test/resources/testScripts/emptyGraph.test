# Tests that an empty graph can be created.

# Create a graph
CreateGraph emptyGraph

# Prints the nodes in the graph
ListNodes emptyGraph