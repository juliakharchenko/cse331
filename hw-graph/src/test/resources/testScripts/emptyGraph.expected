# Tests that an empty graph can be created.

# Create a graph
created graph emptyGraph

# Prints the nodes in the graph
emptyGraph contains: